script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -186,7 +199,7 @@ My tears won't stop.
 [sha:c7277e217c0dedbd53d060bd2af840d5be505ea2]{
 -- Page 258, Offset 4695.
 -- 人を殺したというコトが、ひどく悲しかった。
-I'm distraught at the thought of having killed her.
+The thought of having killed someone is... distraughting.
```

Comment:
I know distraughting is a word but it somehow doesn't show up in dictionaries. A little help please? @rschlaikjer @Avuxo  @yk1512